### PR TITLE
Add precision to scope

### PR DESCRIPTION
**Pull request by AramZS**
Intended to address concerns that the scope requires more precision by adding specificity about intended implementation.

Noting this conflicts with https://github.com/patcg/patwg-charter/pull/49 but may resolve the same objections raised that led to that PR. 

See https://github.com/patcg/patwg-charter/issues/44 to help with understanding this issue.

### CODE REVIEW

File: charter.html [hunk 1]
```diff
@@ -168,8 +168,7 @@ <h2>Scope</h2>
             state.
         </p>
         <p>
-          The Working Group may consider designs that allow user agents for the
-          same user — including non-browser agents, like Operating Systems — to
+          The Working Group may consider designs that allow browser or browser-like user agents (using web technologies) to
```
Comment (martinthomson) [inline charter.html:171]: Is there a need to call out "web technologies" here?  The scope I am thinking of ensures that attribution is possible for a mobile app, which doesn't necessarily use web technology.

Comment (AramZS) [reply charter.html:171]: I think the concern is that the lack of precision could potentially end up resulting in our scope becoming overlarge. Is there a different phrasing you'd suggest or do you think the phrasing without that specific parenthetical is sufficient?

Comment (martinthomson) [reply charter.html:171]: Dropping the parenthetical would help, yes.

Comment (hober) [reply charter.html:171]: I think it's hard to drop that parenthetical & simultaneously keep the scope contained. Though the wording is present earlier in the diff, so maybe there's a way this whole diff could be tweaked to make both Martin and me happy.

**Comment by AramZS** [comment]
I am relaying a comment from a member who has raised formal objections. I do not intend to imply support or opposition to this statement, but just to make their objection on this available for discussion with the member's permission: 

> the proposed text perpetuates the privileged status of the web browser in a field where this is not justified. The web browser is a gateway to the web. Advertising is not a component of that gateway. Adopting a charter which changes this has significant implications for the established business process which we and others have commented on. Apple’s comment stands out as yet another attempt by a web browser vendor to perpetuate the mythical privileged status of the web browser which is the least significant of the priority of constituents. The text change is insufficient to address the FO. We remind you of the CMA's concerns in this regard[^1].

[^1]: https://www.gov.uk/cma-cases/mobile-browsers-and-cloud-gaming

File: charter.html (hunk 1 above)
Comment (eriktaubeneck) [reply charter.html:171]: @hober I fully support assuring that the scope is well contained. That said, I'm potentially not creative enough to what might be unintentionally included by removing the parenthetical (and I'm also unclear on what actually qualifies as "web technologies". Would apps that communicate over the internet (but not in a browser) count?)

Can you provide a couple examples of surfaces/systems which shouldn't be included, but (without the parenthetical) might cause friction in the future as someone views them as included in the scope of the charter?

**Comment by AramZS** [comment]
There is some confusion as to what "browser-like" is intended to mean that would help, if resolved, to resolve this PR.

File: charter.html (hunk 1 above)
Comment (hober) [reply charter.html:171]: > (and I'm also unclear on what actually qualifies as "web technologies". Would apps that communicate over the internet (but not in a browser) count?)

No. The web is only one of the many things that happens over the internet.

> Can you provide a couple examples of surfaces/systems which shouldn't be included, but (without the parenthetical) might cause friction in the future as someone views them as included in the scope of the charter?

You can imagine a browser-like app that doesn't use web technologies. A Gopher browser, for instance, would be a browser-like app that doesn't use web technologies.

Comment (benjaminsavage) [reply charter.html:171]: @hober - I would specifically *want* our charter to encompass both apps and websites - as advertising is shown on both, and purchases happen in both, and connecting the two is critical for measurement.

I'd also like to make sure that to the extent there is any server-side aggregation (e.g. MPC helper parties) that this is also something that would be in-scope for the working group.

Are you *opposed* to the scope of the PAT-WG including apps and MPC helper parties?

Comment (hober) [reply charter.html:171]: > @hober - I would specifically _want_ our charter to encompass both apps and websites - as advertising is shown on both, and purchases happen in both, and connecting the two is critical for measurement.
> 
> I'd also like to make sure that to the extent there is any server-side aggregation (e.g. MPC helper parties) that this is also something that would be in-scope for the working group.
> 
> Are you _opposed_ to the scope of the PAT-WG including apps and MPC helper parties?

I'm opposed to chartering a WG at W3C with a scope broader than W3C's, which is the web. "Some software that maybe sometimes sends data over the internet" is far too broad.

I suspect many or maybe even all aspects of MPC helper parties could be specced at W3C, though it sounds like the kind of thing we might want to do a W3C/IETF split for like we've done in a number of other cases (WebRTC comes to mind).

**Comment by AramZS** [comment]
If we reach consensus on https://github.com/patcg/patwg-charter/pull/56 this will be closing. Accordingly, I am marking this as being under a consensus call

**Pull request closed without merging**